From f44c2c9d0132f170c96ab6a98f608206af048f5c Mon Sep 17 00:00:00 2001
From: Jay Monkman <jay.monkman@freescale.com>
Date: Wed, 3 Apr 2013 17:47:29 -0500
Subject: [PATCH 2449/2463] ENGR00257244 Fixed endless collisions in EPDC

Fixed reversed conditional, causing endless collions in some
situations.

Signed-off-by: Jay Monkman <jay.monkman@freescale.com>
(cherry picked from commit 939f49daf76997e870e20880a0d9e6835d813b74)
---
 drivers/video/mxc/mxc_epdc_fb.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/video/mxc/mxc_epdc_fb.c b/drivers/video/mxc/mxc_epdc_fb.c
index 1122e21..4103498 100644
--- a/drivers/video/mxc/mxc_epdc_fb.c
+++ b/drivers/video/mxc/mxc_epdc_fb.c
@@ -2253,7 +2253,7 @@ static int epdc_submit_merge(struct update_desc_list *upd_desc_list,
 	/* Merged update should take on the earliest order */
 	upd_desc_list->update_order =
 		(upd_desc_list->update_order > update_to_merge->update_order) ?
-		update_to_merge->update_order : upd_desc_list->update_order;
+		upd_desc_list->update_order : update_to_merge->update_order;
 
 	return MERGE_OK;
 }
-- 
1.7.7.4

